### DISCUSSION TITLE
What 🤨?

### DISCUSSION THREAD

**Discussion by PavanLuca** [community/community#63652] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

![Schermata del 2023-07-23 15-58-51](https://github.com/community/community/assets/123581658/a649504b-537a-476a-ac20-685548414299)

**Comment by queenofcorgis** [comment]
Hi @PavanLuca , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by PavanLuca** [reply]
Just a kind of meme probably, since it's weird to read such sentences nowadays.
